smithy-swift-codegen/src/test/kotlin/software/amazon/smithy/swift/codegen/HttpProtocolClientGeneratorTests.kt

```diff
@@ -37,32 +37,35 @@ public class RestJsonProtocolClient: ClientRuntime.Client {
 }
 
```

Comment:
The codegen test expectation below shows how the generated code is different than before.